Automattic/wp-calypso
# Rename FSE subfeature to dotcom fse

#### Changes proposed in this Pull Request 
* Change `full-site-editing-plugin/full-site-editing` to `full-site-editing-plugin/dotcom-fse`

@vindl and I chatted today about the plugin rename, and at least for now, we feel it'll introduce even greater confusion if we rename the plugin. Instead, for now, we'll improve clarity in the codebase by changing the `full-site-editing` subfeature to `dotcom-fse`. This will help us especially when we begin to incorporate core parts of FSE to dotcom.

### Testing instructions
Essentially, just make sure nothing changed with the build. Everything should behave exactly as it did before.

1. Local build should work correctly in a local WordPress install and on dotcom
2. Fse build step in CircleCI should complete and the artifact should work correctly on the sandbox
